src/ops/ops.go

```diff
@@ -59,9 +59,9 @@ type ops struct {
 }
 
 // NewOps return a new ops interface
-func NewOps(logger *logrus.Logger) Ops {
+func NewOps(logger *logrus.Logger, proxySet bool) Ops {
 	cmdEnv := os.Environ()
-	if config.GlobalConfig.HTTPProxy != "" || config.GlobalConfig.HTTPSProxy != "" {
+	if proxySet && (config.GlobalConfig.HTTPProxy != "" || config.GlobalConfig.HTTPSProxy != "") {
```

Comment:
maybe there is no need for config.GlobalConfig.HTTPProxy ?

Reply:
it is needed for the assisted-installer only. but since this code is shared between the installer and the controller - only one requires it.

Reply:
got it , config.GlobalConfig.HTTPProxy  is for installer